common/remoteviewinterface.h

```diff
@@ -85,6 +85,8 @@ public slots:
     void reset();
     void elementsAtReceived(const GammaRay::ObjectIds &ids, int bestCandidate);
     void frameUpdated(const GammaRay::RemoteViewFrame &frame);
+    /// when server receive this signal, obtain a new frame and send it to the client
+    void requestUpdate();
```

Comment:
Hm, I can move it back to server again , that's indeed a performance penalty.
The issue was that I then can't do any Qt5 connect style to a lambda.
With Qt 4 connect style I would need a slot, being *checked* by the unit test framework.